projects/plugins/crm/admin/company/view.page.php

```diff
@@ -594,7 +590,7 @@ function jpcrm_render_company_view_page( $id = -1 ) {
 								// prep link to create a new invoice
 								$new_invoice_url = jpcrm_esc_link( 'create', -1, ZBS_TYPE_INVOICE ) . '&prefillco=' . $company['id'];
 
-								if ( $company_invoices_count > 0 ) {
+								if ( count( $company['invoices'] ) > 0 ) {
```

Comment:
FYI, `$company_invoices_count` still exists. :^)

Reply:
Right, but as I understand it there we want the real count (without deleted invoices). If it was 0 factoring in a deleted invoice, it won't show that deleted invoice under the 'invoices' tab on the company view page.

Reply:
In that case you can use `$company['invoices_count_with_deleted']`. But either way works. :^)

Reply:
Ha... if I have it now, I ought to use it 🤦